telemetry/definitions/commonDefinitions.json

```diff
@@ -788,6 +788,14 @@
             ],
             "passive": true
         },
+        {
+            "name": "aws_loginWithBrowser",
```

Comment:
there is also `aws_openUrl` . 

did we consider adding this as metadat on `aws_validateCredentials` instead? or introduce a `aws_login` which could have a flag that indicates the kind of login?